### PR TITLE
Lose condition

### PR DESCRIPTION
**Pull request by jennatuckerdeveloper**
the characters die, and when the 'you' character dies, the game is over 
with tests

### CODE REVIEW

File: src/Integration.test.js [hunk 1]
```diff
@@ -226,4 +234,76 @@ describe('integration testing', () => {
...
+
+  it('changes the status of one person to "dead" when any number of health scores reaches 0 or less', () => {
+    toNaming()
+    toPacking()
+    toPlaying()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
```
Comment (psbanka) [inline src/Integration.test.js:257]: So you happen to know that doing `onPlay()` five time results in death?

File: src/Integration.test.js [hunk 2]
```diff
@@ -226,4 +234,76 @@ describe('integration testing', () => {
...
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
```
Comment (psbanka) [inline src/Integration.test.js:288]: Similarly, `onPlay()` 9 times results in `You` death? Seems a little brittle a test, doesn't it?

**Review by psbanka — APPROVED** [review]
Looks good!

File: src/Integration.test.js (hunk 2 above)
Comment (jennatuckerdeveloper) [reply src/Integration.test.js:288]: Yes.  It's garbage.  I'll get some advice on how to test it tonight.

**Pull request merged**